.github/pull_request_template.md

```diff
@@ -2,4 +2,9 @@ PR checklist:
 
 - [ ] Documentation is up-to-date
 - [ ] Changelog is up-to-date
+- [ ] Changelog includes:
+  - A reference to a Linear ticket
```

Comment:
what do we do when there's no linear ticket but we want to make a changelog entry?

Reply:
Will update this here and in the docs.